src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -1189,6 +1189,99 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
+                                state.dataHeatBal->Infiltration(Loop).Name);
+            SetupOutputVariable(state,
+                                "Infiltration Standard Density Volume Flow Rate",
+                                OutputProcessor::Unit::m3_s,
+                                state.dataHeatBal->Infiltration(Loop).InfilVdotStdDensity,
+                                OutputProcessor::SOVTimeStepType::Zone,
+                                OutputProcessor::SOVStoreType::Average,
+                                state.dataHeatBal->Infiltration(Loop).Name);
+            SetupOutputVariable(state,
+                                "Infiltration Current Density Volume",
+                                OutputProcessor::Unit::m3,
+                                state.dataHeatBal->Infiltration(Loop).InfilVolumeCurDensity,
+                                OutputProcessor::SOVTimeStepType::Zone,
+                                OutputProcessor::SOVStoreType::Summed,
+                                state.dataHeatBal->Infiltration(Loop).Name);
+            SetupOutputVariable(state,
+                                "Infiltration Standard Density Volume",
+                                OutputProcessor::Unit::m3,
+                                state.dataHeatBal->Infiltration(Loop).InfilVolumeStdDensity,
+                                OutputProcessor::SOVTimeStepType::Zone,
+                                OutputProcessor::SOVStoreType::Summed,
+                                state.dataHeatBal->Infiltration(Loop).Name);
+            SetupOutputVariable(state,
+                                "Infiltration Mass",
+                                OutputProcessor::Unit::kg,
+                                state.dataHeatBal->Infiltration(Loop).InfilMass,
+                                OutputProcessor::SOVTimeStepType::Zone,
+                                OutputProcessor::SOVStoreType::Summed,
+                                state.dataHeatBal->Infiltration(Loop).Name);
+            SetupOutputVariable(state,
+                                "Infiltration Mass Flow Rate",
+                                OutputProcessor::Unit::kg_s,
+                                state.dataHeatBal->Infiltration(Loop).InfilMdot,
+                                OutputProcessor::SOVTimeStepType::Zone,
+                                OutputProcessor::SOVStoreType::Average,
+                                state.dataHeatBal->Infiltration(Loop).Name);
+            SetupOutputVariable(state,
+                                "Infiltration Air Change Rate",
+                                OutputProcessor::Unit::ach,
+                                state.dataHeatBal->Infiltration(Loop).InfilAirChangeRate,
+                                OutputProcessor::SOVTimeStepType::Zone,
+                                OutputProcessor::SOVStoreType::Average,
+                                state.dataHeatBal->Infiltration(Loop).Name);
+
```

Comment:
```
Output:Variable,*,Infiltration Sensible Heat Loss Energy,hourly; !- Zone Sum [J]
Output:Variable,*,Infiltration Sensible Heat Gain Energy,hourly; !- Zone Sum [J]
Output:Variable,*,Zone Infiltration Sensible Heat Loss Energy,hourly; !- HVAC Sum [J]
Output:Variable,*,Zone Infiltration Sensible Heat Gain Energy,hourly; !- HVAC Sum [J]
```
Here from an example rdd, the newly added object level lines (the first two) has a notation of "Zone Sum [J]", which seems to be a little bit confusing when comparing it with the zone level report together. I realized the we may have not a better category put it in rather than "Zone Sum[J]---but any other way to demonstrate the differences? Just a thought.

Reply:
Good question. Actually I didn't know whether it should be `Zone Sum` or `System Sum`. I was following the `People` reporting method so I put the reporting timestep as `Zone`, but honestly I don't know the nuance. Could @mjwitte and @rraustad clarify a bit?

Reply:
Regarding Zone vs HVAC for output variables, the output processor updates output variables and writes out time series values at the end of either the zone timestep or the system timestep. These updates include summing or averaging as needed depending on the selected reporting frequency. This is specified in the `SetupOutputVariable` function as either `OutputProcessor::SOVTimeStepType::System` or  `OutputProcessor::SOVTimeStepType::Zone`. There are only two choices and depending on where the value is finalized for each time step, there's only one correct answer. A variable that updates every system time step will report incorrect values if it's set up as `Zone`. I suppose a variable that updates at the zone timestep wouldn't be wrong if it was set up as `System` but it would exercise the output processor more than necessary and would report the same value for every HVAC timestep within a given zone timestep.

If a variable changes during HVAC timesteps then System should be specified.  Infiltration is updated during the HVAC simulation, and the existing infiltration outputs use `System`, so these new ones should also be `System`. Other internal gains, like People, are only updated at the zone time step, so that's why they use `Zone`.

Reply:
Thanks @mjwitte for the clarification, very helpful. 👍